pkg/scheduler/objects/nodesorting.go

```diff
@@ -42,33 +48,79 @@ func (fairnessNodeSortingPolicy) PolicyType() policies.SortingPolicy {
 	return policies.FairnessPolicy
 }
 
-func (binPackingNodeSortingPolicy) ScoreNode(node *Node) float64 {
-	// choose most loaded node first
-	return resources.LargestUsageShare(node.GetAvailableResource())
+func absResourceUsage(node *Node, weights *map[string]float64) float64 {
```

Comment:
we need a better name for this method, this is calculating the node score, maybe we can call it `calculateNodeScore()`.

Reply:
It isn't calculating a score (though the current scoring algorithms use it in a similar manner). It's calculating the % of the node's resources that are in-use (subject to weighting).

Reply:
this looks like scoring the node to me because it applies the weights and returns a float per node. but if you think the current naming is good, that's fine I am OK with that.